src/Symfony/Component/ValueExporter/Exporter/AbstractValueExporter.php

```diff
@@ -72,23 +79,19 @@
         foreach ($formatters as $formatter) {
             if (is_array($formatter)) {
                 $priority = (int) $formatter[1];
-                $formatter = $formatter[0];
+                $formatterClass = $formatter[0];
```

Comment:
Indeed, my first draw used `list`. I end up using that notation because I find it more readable, especially when casting to int :)
Since the priority in the value used to sort with `arsort` I want to be sure some `'10'` string doesn't mess with the order, do you think that's only paranoia 😨?